kura/org.eclipse.kura.wire.component.provider/src/main/java/org/eclipse/kura/internal/wire/publisher/CloudPublisher.java

```diff
@@ -431,15 +364,16 @@ private void publish(final List<WireRecord> wireRecords) {
             for (final WireRecord dataRecord : wireRecords) {
```

Comment:
I beg your pardon. It's not possible. We have to send every record per payload. 👍